### DISCUSSION TITLE
First UC instal on Mac

### DISCUSSION THREAD

**Discussion by akasaka99** [Eloston/ungoogled-chromium#1577] [Q&A]
Hi, I am new to UC so sorry i'm my questions are basic.
I just installed latest UC 92 on my Mac with brew.
1. Where can I check if my version is up to date ("About Chromium" does not seem to do any version checks)?
2. In the future, I wonder whether I'll need re-install it with each new update manually?
3. Will I have to reinstall all the Extensions or the new version will pick-up all the existing extension?
Many thanks for help!

**Comment by networkException** [comment]
1 & 2. Brew is a package manager that handles updates for you. If you run `brew upgrade` it will automatically check if installed software is up to date and install a newer version if available
3. Replacing the chromium program leaves all your configuration including extensions, history, passwords and even opened tabs (if you enable to reopen the previous tabs after closing chromium) untouched, they reside in deticated directories

**Reply by akasaka99** [reply]
Many thanks! So it means that I'll need to run "brew upgrade" from time to time. So if I dont do that, UC wont upgrade. Do I understand you?

**Reply by networkException** [reply]
Yes exactly

**Reply by akasaka99** [reply]
Understood. Thanks !!

**Comment by akasaka99** [comment]
One last question, is there's a place where there's an announcement when a new UC build is released?

**Reply by kramred** [reply]
The macOS brew cask formula has been using by GitHub Action builds for some time now. So you could watch [that repo](https://github.com/kramred/ungoogled-chromium-macos/) for new macOS releases and be notified via GitHub.

The [main ungoogled-chromium repo](https://github.com/Eloston/ungoogled-chromium), i.e. this one where we are right now, also has releases when a new version is available but the binaries/builds are then somewhere else, e.g. linked in the [ungoogled-chromium-binaries repo](https://github.com/ungoogled-software/ungoogled-chromium-binaries), or in the [platform specific repos](https://github.com/ungoogled-software/)

**Reply by networkException** [reply]
Yes, you can subscribe to new releases right here on GitHub!

Choose custom in the Watch dropdown menu
![image](https://user-images.githubusercontent.com/42888162/127389249-14485b61-a9e4-4ea3-b82c-62023704d434.png)
then select releases
![image](https://user-images.githubusercontent.com/42888162/127389278-937199c5-0433-40ff-a0c3-23b4cfb544a7.png)

**Reply by akasaka99** [reply]
I must admit i'm a little bit lost.
1. I installed UC on my Mac with Homebrew and I got the version: 92.0.4515.107 from https://github.com/Eloston/ungoogled-chromium
2. If as suggested I look at https://github.com/ungoogled-software/ which brings me to Mac repos, I would have gotten the version: 91.0.4472.77
=> So one would want to have the latest version from 1) 92.xxx version as opposed to 2) 91.xxx version? 

Or is it a little bit like on Linux were 91.xxx would be an "LTS"-equivalent more stable version as opposed to 92.xxx a normal rolling release version ?

**Reply by networkException** [reply]
The brew release is based on [this repository](https://github.com/kramred/ungoogled-chromium-macos/) which is using the changes from this [pull request](https://github.com/Eloston/ungoogled-chromium/pull/1573). Essentially Eloston just hasn't found time to update this repository. And no chromium 91 is not an LTS release, its the previous stable release and will not recieve updates going forward
